src/terminatesteadystate.jl

```diff
@@ -0,0 +1,20 @@
+
+# Default test function
+# Terminate when all derivatives fall below a threshold, with early exit
+function allDerivPass(derivs, tol)
```

Comment:
You can use the Julia function `any` for this. `any` will iterate and exit early if any are found true. I think we should use a relative tolerance too, like in this issue ( https://github.com/JuliaDiffEq/DifferentialEquations.jl/issues/228 ).